freer-extras/freer-extras.cabal

```diff
@@ -109,4 +109,4 @@ test-suite freer-extras-test
     , semigroups
     , sqlite-simple
     , tasty
-    , tasty-hedgehog
+    , tasty-hedgehog1
```

Comment:
Typo?

Reply:
fuuuuuuu, this ended up in CHaP as well :facepalm: yes, it's a typo